docs/release-notes.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: docs
+title: "Release notes"
+permalink: /docs/release-notes
+---
+
+# Release notes
+
+## Microk8s 1.16 - 26 September 2019
+
+**MicroK8s** is a Kubernetes<sup>&reg;</sup> cluster delivered as a single snap
+package - it can be installed on any Linux distribution which supports
+[snaps][]. MicroK8s is small and simple to install and is a great way to stand
+up a cluster quickly for development and testing. Try it on your laptop!
+
+```bash
+snap install microk8s --classic
```

Comment:
As we are having all releases here we should inclue the `--channel=1.16/stable` because without the channel you will not get the right channel in a few months.

Reply:
ok